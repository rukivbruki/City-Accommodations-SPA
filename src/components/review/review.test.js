import React from "react";
import renderer from "react-test-renderer";
import Review from "./review";

it(`Review component render correct`, () => {
  const reviewMockData = {
    avatarUrl: `img/apartment-01.jpg`,
    comment: `Hello world`,
    date: `11 April`,
    rating: 4.2,
    userName: `Serhii`
  };

  const review = renderer
    .create(<Review {...reviewMockData}/>)
    .toJSON();

  expect(review).toMatchSnapshot();
});
