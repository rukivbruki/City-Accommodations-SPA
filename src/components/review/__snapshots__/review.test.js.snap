// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Review component render correct 1`] = `
<li
  className="reviews__item"
>
  <div
    className="reviews__user user"
  >
    <div
      className="reviews__avatar-wrapper user__avatar-wrapper"
    >
      <img
        alt="Serhii avatar"
        className="reviews__avatar user__avatar"
        height="54"
        src="img/apartment-01.jpg"
        width="54"
      />
    </div>
    <span
      className="reviews__user-name"
    >
      Serhii
    </span>
  </div>
  <div
    className="reviews__info"
  >
    <div
      className="reviews__rating rating"
    >
      <div
        className="reviews__stars rating__stars"
      >
        <span
          style={
            Object {
              "width": "84.00000000000001%",
            }
          }
        />
        <span
          className="visually-hidden"
        >
          Rating
        </span>
      </div>
    </div>
    <p
      className="reviews__text"
    >
      Hello world
    </p>
    <time
      className="reviews__time"
      dateTime="2019-04-24"
    >
      11 April
    </time>
  </div>
</li>
`;
